cmd/process-agent/app/version.go

```diff
@@ -0,0 +1,44 @@
+package app
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	"io"
+	"os"
+	"runtime"
+
+	"github.com/DataDog/datadog-agent/pkg/serializer"
+	"github.com/DataDog/datadog-agent/pkg/version"
+)
+
+// VersionCmd is a command that prints the process-agent version data
+var VersionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Print the version info",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return WriteVersion(os.Stdout)
+	},
+	SilenceUsage: true,
+}
+
+// versionString returns the version information filled in at build time
+func versionString(v version.Version) string {
+	return fmt.Sprintf(
+		"Agent %s - Commit: %s - Serialization version: %s - Go version: %s",
+		v.GetNumberAndPre(),
+		v.Commit,
+		serializer.AgentPayloadVersion,
+		runtime.Version(),
+	)
```

Comment:
What do you think of using the same pattern from the core agent and printing the output with colors?
https://github.com/DataDog/datadog-agent/blob/e081bed84866686eeab56a98da5ff9b4b9f03ded/cmd/agent/app/version.go#L28-L45

Reply:
Added!